posts/hello-microservice-deployment-part-2.md

```diff
@@ -0,0 +1,358 @@
+```
+
+This will create a three node cluster called hello-timber. It might take a few minutes. Each node is a compute instance (VM) managed by Google. I've chosen here to put the nodes in west Europe. If you want to see a full list of available zones you can use the command `gcloud compute zones list`. Alternatively you can refer to [this document](https://cloud.google.com/compute/docs/regions-zones/) for a list of available zones.
+
+In general it is good practice to choose a zone that is close to whatever clients are consuming your application. Although it is possible to deploy an application on multiple clusters in multiple zones around the world, that is waaaay outside the scope of this tutorial.
+
+Once your cluster has been created successfully you should see something like this:
+
+```
+WARNING: Currently node auto repairs are disabled by default. In the future this will change and they will be enabled by default. Use `--[no-]enable-autorepair` flag  to suppress this warning.
+WARNING: Starting in Kubernetes v1.10, new clusters will no longer get compute-rw and storage-ro scopes added to what is specified in --scopes (though the latter will remain included in the default --scopes). To use these scopes, add them
+ explicitly to --scopes. To use the new behavior, set container/new_scopes_behavior property (gcloud config set container/new_scopes_behavior true).
+Creating cluster hello-timber...done.
+Created [https://container.googleapis.com/v1/projects/timber-tutorial/zones/europe-west1-d/clusters/hello-timber].
+To inspect the contents of your cluster, go to: https://console.cloud.google.com/kubernetes/workload_/gcloud/europe-west1-d/hello-timber?project=timber-tutorial
+kubeconfig entry generated for hello-timber.
+NAME          LOCATION        MASTER_VERSION  MASTER_IP      MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
+hello-timber  europe-west1-d  1.8.10-gke.0    35.205.54.147  n1-standard-1  1.8.10-gke.0  3          RUNNING
+```
+To see the individual nodes:
+
+```
+gcloud compute instances list
+```
+
+This will output something like:
+
+```
+NAME                                         ZONE            MACHINE_TYPE   PREEMPTIBLE  INTERNAL_IP  EXTERNAL_IP     STATUS
+gke-hello-timber-default-pool-a471ab46-c0q8  europe-west1-d  n1-standard-1               10.132.0.3   35.205.110.255  RUNNING
+gke-hello-timber-default-pool-a471ab46-jv7n  europe-west1-d  n1-standard-1               10.132.0.4   35.189.247.55   RUNNING
+gke-hello-timber-default-pool-a471ab46-mt54  europe-west1-d  n1-standard-1               10.132.0.2   35.190.195.64   RUNNING
+```
+
+Achievement unlocked: K8s cluster!
+
+### Upload your Docker image
+
+We have a docker image on our local machine. We need to make sure it is available to our cluster. This means we need to put our image into a [container registry](https://cloud.google.com/container-registry/) that Google can access. Google cloud has a container registry built in.
```

Comment:
Write something like 'If you don't already have this, you might have missed [my last article](LINK) where I walked you through creating one'